tests/README.md

```diff
@@ -36,6 +36,12 @@ environment variable. For example:
 IPA_SSH_PASSWORD=<ipaserver_ssh_password> IPA_SERVER_HOST=<ipaserver_host_or_ip> pytest
 ```
 
+If you want, or ned to to set the Python interpreter to use, you must set `IPA_PYTHON_PATH`
```

Comment:
s/ned/need/g 

if you want or need to set the ...

Reply:
Fixed